guides/v2.2/frontend-dev-guide/css-topics/using-fonts.md

```diff
@@ -42,8 +43,18 @@ To ensure the stability of your customizations and prevent upgrades from overwri
         src: url('../fonts/opensans/light/opensans-300.eot?#iefix') format('embedded-opentype'), url('../fonts/opensans/light/opensans-300.woff2') format('woff2'), url('../fonts/opensans/light/opensans-300.woff') format('woff'), url('../fonts/opensans/light/opensans-300.ttf') format('truetype'), url('../fonts/opensans/light/opensans-300.svg#Open Sans') format('svg');
         font-weight: 300;
         font-style: normal
+        font-display: swap;
     }
     ```
+    
+    `@font-display: swap` is declared by default for Magento Blank theme in `app/design/frontend/Magento/blank/web/css/source/_typography.less`.
+    
+    Fallback web fonts that are used by default in Magento are located in `lib/web/css/source/lib/variables/_typography.less`.
+    
+### `<font>` head type
+
+`<font>` node is added to HTML `<head>` type for layout in `lib/internal/Magento/Framework/View/Layout/etc/head.xsd`. All resources added with `<font>` node will be downloaded with `preload` html attribute. Since preload does not use downloaded resource, you should ensure that this resource is used in CSS or HTML DOM.
```

Comment:
Hi @adifucan  I am a bit confused on this paragraph. What do you mean 'preload does not use downloaded resources'? What does it use? Which resource should we ensure is used?

Reply:
I mean that preload attribute just specifies resources that your page will need very soon, which you want to start loading early in the page lifecycle. This ensures they are available earlier.
Here I mean that fonts will be preloaded now and you should ensure that they are used in CSS/HTML.

Reply:
Maybe it's not obligatory to mention about "Since preload does not use downloaded resource, you should ensure that this resource is used in CSS or HTML DOM." here because preload functionality works in this way.